clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/ArraysTests.scala

```diff
@@ -1192,4 +1193,69 @@ trait ArraysTests { self: BaseIntegrationSpec =>
       storedDocument1.unsafeGet[String]("notes") should be("I'm a note")
     }
   }
+
+  it should "amend documents with new metadata" in {
+
+    val metadataFile: File = File
+      .newTemporaryFile()
+      .overwrite(
+        ArraysMetadata(
+          sampleAlias = Some("amended_sample_alias"),
+          chipType = Some("amended_chip_type")
+        ).asJson.pretty(implicitly)
+      )
+      .deleteOnExit()
```

Comment:
can you use better files here to remove this?

Reply:
Actually, scratch that. The for-comprehension needs to be of type `Future`, not `ManagedResource`